packages/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -1,11 +1,10 @@
 import * as React from 'react';
-import { usePopperMouseTarget } from '@fluentui/react-positioning';
-import type { ComponentProps, ComponentState } from '@fluentui/react-utilities';
-import type { PositioningShorthand } from '@fluentui/react-positioning';
-import type { MenuListProps } from '../MenuList/index';
-import type { MenuContextValue } from '../../contexts/menuContext';
+import { ComponentProps, ComponentState } from '@fluentui/react-utilities';
+import { usePopperMouseTarget, PositioningShorthand } from '@fluentui/react-positioning';
+import { MenuListCommons } from '../MenuList/index';
+import { MenuContextValue } from '../../contexts/menuContext';
 
-interface MenuCommons extends MenuListProps {
+interface MenuCommons extends MenuListCommons {
```

Comment:
why did you extend from `MenuListCommons` here ?

Reply:
Is it breaking something? Should I restore it? IMO it makes sense

Reply:
Yh let's keep it then